packages/@react-aria/overlays/src/useOverlayPosition.ts

```diff
@@ -12,29 +12,41 @@
 
 import {calculatePosition, PositionResult} from './calculatePosition';
 import {HTMLAttributes, RefObject, useEffect, useState} from 'react';
-import {Placement, PlacementAxis} from '@react-types/overlays';
+import {Placement, PlacementAxis, PositionProps} from '@react-types/overlays';
 import {useLocale} from '@react-aria/i18n';
 
-export interface PositionProps {
-  placement?: Placement,
-  containerPadding?: number,
-  offset?: number,
-  crossOffset?: number,
-  shouldFlip?: boolean,
-  boundaryElement?: HTMLElement,
-  isOpen?: boolean
-}
-
 interface AriaPositionProps extends PositionProps {
+  /**
+   * Element that that serves as the positioning boundary.
+   * @default document.body
+   */
+  boundaryElement?: HTMLElement,
+  /**
+   * The ref for the element which the overlay positions itself with respect to.
+   */
   targetRef: RefObject<HTMLElement>,
+  /**
+   * The ref for the overlay element.
+   */
   overlayRef: RefObject<HTMLElement>,
+  /**
+   * TODO come up with a good description
+   * @default overlayRef
+   */
   scrollRef?: RefObject<HTMLElement>,
```

Comment:
Need some help coming up with a description of this

Reply:
it seems distinct from the overlay ref though since a user can technically provide some other ref instead. I don't know when that case would happen but its the reason why I'm not super sure how to describe it.

Reply:
"A ref for the scrollable region within the overlay." something like that?